app/scripts/models/reliers/relier.js

```diff
@@ -101,6 +102,19 @@ define([
     },
 
     /**
+     * Check if the relier is Sync for Firefox Desktop
+     */
+    isChooseWhatToSyncWeb: function (uniqueUserId, locationParams, able) {
```

Comment:
Can this be moved to the fx-desktop relier since it's pretty specific to that relier?